source/blog/2018-06-29-ember-3-2-released.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Ember 3.2 and 3.3 Beta Released
+author: Ricardo Mendes, Melanie Sumner & Kenneth Larsen
+tags: Releases, 2018, 3, 3.2, 3.3
+responsive: true
+---
+
+Today the Ember project is releasing version 3.2.0 of Ember.js, Ember Data, and Ember CLI.
```

Comment:
Should we explain why we're not including 3.3 Beta notes?

Reply:
Dunno. I just don't have the time, and things have been delayed enough 😢

Reply:
okay. let's just leave it out for now. :)